src/integrations/Elastic.Apm.AspNetFullFramework/Extensions/SoapRequest.cs

```diff
@@ -20,6 +20,8 @@ internal static class SoapRequest
 		private const string SoapActionHeaderName = "SOAPAction";
 		private const string ContentTypeHeaderName = "Content-Type";
 		private const string SoapAction12ContentType = "application/soap+xml";
+		private const int MaxRequestBodySizeInBytesToCapture = 1024 * 10;
```

Comment:
This is a fairly arbitrary limit and I'm totally open to change it if we have any better number. I just naively think that capturing something bigger than this doesn't make a lot of sense, so I choose this limit.

Reply:
10kb seems reasonable, even if it is SOAP 🙈